tests/petsc_smoketest.cpp

```diff
@@ -96,7 +96,7 @@ PetscErrorCode ComputeRHS(KSP ksp, Vec b, void* ctx)
   PetscFunctionBeginUser;
   ierr = KSPGetDM(ksp, &da);
   CHKERRQ(ierr);
-  ierr = DMDAGetInfo(da, 0, &M, &N, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
+  ierr = DMDAGetoptions(da, 0, &M, &N, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
```

Comment:
DMDAGetOptions

Reply:
Nevermind I guess this isn't ours or if it is I'm too tired to find it.

Reply:
Yep, it's not ours.